### PR TITLE
added-authentication-page

### PR DESCRIPTION
**Pull request by anshtripathi01**
**Please Review**

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
Remove logs from prioject rest looks good

**Comment by SRUJANA-PENUGONDA13** [comment]
Code is very clean and easy to understand.

File: src/pages/Auth/Signup.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    <i class="fa-solid fa-eye-slash" id="togglePassword"></i>
+                    <input onChange={(e) => setPassword(e.target.value)} class="form-input" type="password" placeholder="Create your password" />
+                    <label class="form-label" for="password" >Confirm Password</label>
+                    <input class="form-input" type="password" placeholder="Type your password again" />
+                    <span className="form-checkbox">
+                        <input type="checkbox" />
+                        <label for="checkbox">I agree with terms</label>
+                    </span>
+                    <span className="form-btn">
+                        <button className="btn btn-solid-primary" onClick={signupHandler}>Sign Up</button>
```
Comment (JatuFaique) [inline src/pages/Auth/Signup.js:67]: A suggestion, I think setting the button as type submit and using the signupHandler on form submit would be a more semantic way to put it. still, it works good.

**Review by JatuFaique — COMMENTED** [review]
Liked the authentication implementation, and the use of CL components is clean.
Looks good to merge.

**Comment by JatuFaique** [comment]
Looks good to merge.

File: src/pages/Auth/Signup.js (hunk 1 above)
Comment (anshtripathi01) [reply src/pages/Auth/Signup.js:67]: ok thank you

**Pull request merged**